pysac/mhs_atmosphere/mhs_atmosphere.py

```diff
@@ -0,0 +1,290 @@
+# -*- coding: utf-8 -*-
```

Comment:
If this is the actual script that generates the atmosphere (which it looks like it is), it should be in `examples` rather than in the library.

The fact that it is called the same as the submodule it is in and is not imported is very confusing!